recipes/default.rb

```diff
@@ -151,16 +176,16 @@
   cookbook_file File.join(node['android-sdk']['scripts']['path'], android_helper_script) do
     source android_helper_script
     owner node['android-sdk']['scripts']['owner']
-    group node['android-sdk']['scripts']['group']
-    mode 0755
+    group node['android-sdk']['scripts']['group'] unless mac_os_x?
+    mode '0755'
   end
 end
 %w(android-update-sdk).each do |android_helper_script|
   template File.join(node['android-sdk']['scripts']['path'], android_helper_script) do
     source "#{android_helper_script}.erb"
     owner node['android-sdk']['scripts']['owner']
-    group node['android-sdk']['scripts']['group']
-    mode 0755
+    group node['android-sdk']['scripts']['group'] unless mac_os_x?
```

Comment:
I had some trouble to set properly group on mac, but now it doesn't occurs. I will remove `unless mac_os_x?` today and recommit to branch.